packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -28,6 +37,53 @@ const OpenQueueCard = ({ queue }: OpenQueueCard) => {
       }
     >
       <h1>{queue.room}</h1>
+
+      {queue.notes && (
+        <div>
+          <div>
+            <b>Staff Notes:</b>
+            {isTA && (
+              <EditOutlinedFloatedRight
+                onClick={() => {
+                  setEditingNotes(true);
+                }}
+              />
+            )}
+          </div>
+          {editingNotes ? (
+            <Input
+              defaultValue={queue.notes}
+              onPressEnter={(value) => {
+                API.queues.updateNotes(queue.id, (value.target as any).value);
+                queue.notes = (value.target as any).value;
+                setEditingNotes(false);
+              }}
+            />
+          ) : (
+            <p>{queue.notes}</p>
+          )}
+        </div>
+      )}
+      {!queue.notes &&
+        (editingNotes ? (
+          <Input
+            defaultValue={queue.notes}
```

Comment:
since `queue.notes` is an optional and this is the case where `queue.notes` is undefined, `defaultValue` could be left out or just made blank